src/Workspaces/CSharp/Portable/Formatting/Rules/SpacingFormattingRule.cs

```diff
@@ -321,6 +321,12 @@ public override AdjustSpacesOperation GetAdjustSpacesOperation(SyntaxToken previ
                 return CreateAdjustSpacesOperation(0, AdjustSpacesOption.ForceSpaces);
             }
 
+            // Always put a space in the var form of deconstruction-declaration
+            if (currentKind == SyntaxKind.OpenParenToken && currentToken.Parent.Kind() == SyntaxKind.VariableDeconstructionDeclarator && previousToken.Kind() == SyntaxKind.IdentifierToken && previousToken.ValueText == "var")
```

Comment:
I'm slightly concerned that `Parent` can be null here, if the formatting API is asked to format just a random stream of tokens.

@heejaechang does that sound possible to you?  @jcouv if so, you might consider using `currentToken.IsParentKind(SyntaxKind.VariableDeconstructionDeclarator)` instead.